packages/tools/aml-dtbtools/package.mk

```diff
@@ -0,0 +1,35 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2016 Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="aml-dtbtools"
+PKG_VERSION="cce100f"
+PKG_ARCH="any"
+PKG_LICENSE="free"
+PKG_SITE="https://github.com/Wilhansen/aml-dtbtools"
+PKG_URL="${PKG_SITE}/archive/${PKG_VERSION}.tar.gz"
```

Comment:
sticking with conventions this should be `PKG_URL="https://github.com/Wilhansen/aml-dtbtools/archive/${PKG_VERSION}.tar.gz"`

Reply:
I know that is the convention, but I think that @kszaq way should be the new convention. Now that we are using github as preferred source, makes more easy use another repository, avoid the mistake of changing the PKG_SITE and not the PKG_URL or vice-versa.

Reply:
This is why PR's have reviews so mistakes don't get made :) I don't think this is a good idea, it would add fragmentation and only 300/700 packages use github.

Reply:
PKG_SITE is not necessarily github even if the download comes from github. Proper fix would be a special var for gh archives. But not too sure we should add complexity to that.